src/test/java/jewellery/inventory/helper/ProductTestHelper.java

```diff
@@ -91,19 +92,19 @@ public static ProductResponseDto getReturnedProductResponseDto(
     productResponseDto.setProductsContent(null);
     productResponseDto.setContentOf(null);
     productResponseDto.setDescription("description");
-    productResponseDto.setSalePrice(1000);
+    productResponseDto.setSalePrice(BigDecimalUtil.getBigDecimal("1000"));
     productResponseDto.setPartOfSale(null);
     productResponseDto.setCatalogNumber("Catalog Number");
     productResponseDto.setProductionNumber("Production Number");
-    productResponseDto.setDiscount(0);
+    productResponseDto.setDiscount(new BigDecimal("0"));
```

Comment:
The regex I used to find/replace for some reason did not catch BigDecimal("0"). I have no explanation why. Will fix.

Reply:
Fixed but instead of using the utility method I used the recommended way - BigDecimal.ZERO. That is specifically for where we had "new BigDecimal("0")".